assets/src/amp-story-editor-blocks.js

```diff
@@ -179,35 +178,37 @@ const wrapperWithSelect = compose(
  * @param {Object} BlockListBlock BlockListBlock element.
  * @return {Function} Handler.
  */
-const addWrapperProps = createHigherOrderComponent(
```

Comment:
@swissspidy Let me know if there was a reason for using `createHigherOrderComponent` together with `wrapperWithSelect`, I might be not aware of something.

Reply:
Looks like an artefact from when I was shuffling around all the components.